interface/main/finder/dynamic_finder.php

```diff
@@ -61,7 +61,7 @@
     $orderjson .= "[\"$colcount\", \"" . addcslashes($colorder, "\t\r\n\"\\") . "\"]";
     ++$colcount;
 }
-$loading = "<i class='fa fa-sync fa-2x fa-spin'></i>";
+$loading = "<div class='spinner-border spinner-border-sm' role='status'><span class='sr-only'>Loading...</span></div>";
```

Comment:
Ur gonna need to add the xlt translation to the "loading" text

Reply:
OK, Update commit :)